packages/platform/atoms/src/lib/filterAndFormatTimezones.ts

```diff
@@ -0,0 +1,20 @@
+export const filterAndFormatTimezones = (
+  availableTimezones?: { city: string; timezone: string }[],
```

Comment:
I think this actually should be 2 separate functions 1 `filterAndFormatTimezonesFetched` and `filterAndFormatTimezonesProps` - if a function has an optional parameter it means that it probably can be split. And then in `function Timezone` if we have timezones from props then convert props timezone from 1 function and if not then format fetched timezones using another.